src/ray/core_worker/core_worker.cc

```diff
@@ -908,7 +908,8 @@ Status CoreWorker::Seal(const ObjectID &object_id, bool pin_object,
 }
 
 Status CoreWorker::Get(const std::vector<ObjectID> &ids, const int64_t timeout_ms,
-                       std::vector<std::shared_ptr<RayObject>> *results) {
+                       std::vector<std::shared_ptr<RayObject>> *results,
+                       bool plasma_objects_only) {
```

Comment:
What is this change for?

Reply:
the IO worker does not have the ownership info, so it will fail to fetch these objects. specify plasma objects only gets rid of this issue

Reply:
Can't we just not try to spill non-plasma objects?

Reply:
we did not spill non-plasma objects, that is why we need `plasma_objects_only` for `CoreWorker::Get`. Otherwise we will have to pass ownership info to IO workers, which is non-trivial.

basically without `plasma_objects_only`, `CoreWorker::Get` will be pended forever for plasma objects. I actually think this is a flaw of `CoreWorker::Get`, because we do not need ownership control if the plasma objects are in the same node.